local-config.yml

```diff
@@ -53,4 +53,4 @@ utils:
     ping: passive
     flood-ping: passive
     jitter: passive
-    req-res-delay-or-monitoring: passive
\ No newline at end of file
+    req-res-delay-or-monitoring: passive
```

Comment:
This new line might be a mistake. Please fix this.

Reply:
It was already there I think. Are you sure you want me to remove this? @Harkishen-Singh